pom.xml

```diff
@@ -16,7 +16,7 @@
         <!-- check micrometer.version vertx-micrometer-metrics consumes before bumping up -->
         <micrometer.version>1.1.0</micrometer.version>
         <junit-jupiter.version>5.7.0</junit-jupiter.version>
-        <uid2-shared.version>2.5.0-6429c70acc</uid2-shared.version>
+        <uid2-shared.version>2.5.6-SNAPSHOT</uid2-shared.version>
```

Comment:
Should be reverted for the commit

Reply:
Yeah it will need to be changed to the real release when it happens